### ISSUE TITLE
dockerfile/ubuntu removed in docker hub registry?

### ISSUE THREAD

**Issue by wbingli** [dockerfile/ubuntu#10]
```
docker pull dockerfile/ubuntu
Pulling repository dockerfile/ubuntu
FATA[0001] Error: image dockerfile/ubuntu:latest not found
```

Looks all dockerfile images cannot be found from docker hub.

**Comment by corpulent** [comment]
I am getting the same thing:

```
root@XXX:~# sudo docker pull dockerfile/ubuntu
Pulling repository dockerfile/ubuntu
FATA[0000] Error: image dockerfile/ubuntu:latest not found 
```

My solution was to run this `docker build -t="dockerfile/ubuntu" github.com/dockerfile/ubuntu`  but what about all the other layers that are depending on dockerfile/ubuntu,  like dockerfile/nginx?  I have to install all dependencies manually in order.

**Comment by ghost** [comment]
https://registry.hub.docker.com/u/dockerfile/ubuntu/ is redirecting me to https://registry.hub.docker.com/u/library/ubuntu/

**Comment by fabricionaweb** [comment]
+1 same problem

**Comment by dbu** [comment]
it looks like the whole organisation moved to https://github.com/docker-library/ - is that the case? then maybe updating the github.io page and the readme in the repositories would help...

**Comment by bennekrouf** [comment]
+1

**Comment by doctapp** [comment]
+1

**Comment by andykamto** [comment]
+1 same problem

**Comment by interair** [comment]
+1

**Comment by joelesalas** [comment]
+1

**Comment by shaunc** [comment]
+1 ... um and _everything_ has to be updated that uses this? (was just trying to pull dockerfile/mongodb...) Seems like they should add alias. Hard name switch is harsh.

**Comment by jvanoosterom** [comment]
this move below indeed breaks a lot ...:
http://blog.docker.com/2015/03/updates-available-to-popular-repos-update-your-images/

**Comment by sushilshah** [comment]
+1 
sudo docker pull dockerfile/mongodb
Pulling repository dockerfile/mongodb
Error: image dockerfile/mongodb:latest not found

**Comment by matthewmueller** [comment]
+1

**Comment by brandtabbott** [comment]
+1

**Comment by phoenix888** [comment]
+1

**Comment by allnash** [comment]
- 1

**Comment by frogstarr78** [comment]
+1

**Comment by wkubota** [comment]
+1

**Comment by lukeblechl** [comment]
+1

**Comment by amalagaura** [comment]
WTF all docker images are broken? I can't use the java image either.

**Comment by MarkusPfundstein** [comment]
is there anyone out there from Docker that actually cares about this?

**Comment by amalagaura** [comment]
@MarkusPfundstein I had to search around, but they do exist in other places. They got rename/moved to docker hub I believe.

**Comment by robmcfeely** [comment]
If you read @jvanoosterom comment you will see that they have just moved.  instead of **dockerfile/ubuntu** its just **ubuntu**.  So **docker pull ubuntu**

**Comment by strarsis** [comment]
+1

I try to build nginx official docker image Dockerfile by my own, 
but ubuntu:latest, the official image from Dockerhub, isn't sufficient for building,
the `add-apt-repository` shell script/command is missing (not included in the image):
https://github.com/dockerfile/nginx/blob/master/Dockerfile#L12

Edit: Package '[software-properties-common](http://packages.ubuntu.com/search?searchon=contents&keywords=add-apt-repository&mode=&suite=wily&arch=any)' provides this command, I had to install it first.
As the nginx Dockerfile doesn't have to install that package before invoking this command - 
one must assume the base image deviates from ubuntu in Dockerhub, it is a different one!

Edit: Apparently it is a github repository that has been formerly used for the official nginx repository: 
https://github.com/dockerfile/nginx
The nginx repository links to another github repository: https://github.com/nginxinc/docker-nginx/blob/a8b6da8425c4a41a5dedb1fb52e429232a55ad41/Dockerfile

See https://github.com/dockerfile/nginx/issues/28 .

**Comment by raskolnikov7** [comment]
I got the same error 
Error: image dockerfile/ubuntu:latest not found

**Comment by wutingbupt** [comment]
no one fix?

**Comment by robmcfeely** [comment]
YES ITS WORKS (at least for the basic case) read my last comment.  For dependant dockerfiles you will prob have to update those scripts

**Comment by viclix10** [comment]
Replace 
**docker pull dockerfile/ubuntu**
with
**docker pull ubuntu**

Similarly for Dockerfile, Pull base image, replace
**FROM dockerfile/ubuntu**
with
**FROM ubuntu**

**Comment by ngie-eign** [comment]
Too bad they didn't just fork/rename the repo.
Ubuntu should really archive this at this point. There are examples out there that are still using this repo (example: https://github.com/kennethklee/docker-firefox ), and the OS (14.04) is woefully out of date/insecure at this point in time.
Guess I'll reach out on launchpad about archiving the repo/closing out all of the issues.
